File: 11/INFORMATIKA/11KOM2/AP/LAT01/Soal02.java
import java.util.*;

public class Soal02{

    public static void main(String[] args)
    {
        ArrayList<String> petList = new ArrayList<String>(Arrays.asList("Dog"));
        petList.add("Cat");
        petList.add("Fish");
        petList.add(1, "Golden Bandicoot");
        petList.remove(0);

        System.out.println(String.join(", ", petList));
    }
}

File: 10/INFORMATIKA/C++/10-7/07/soal02.cpp
#include <iostream>
#include <vector>
using namespace std;

int main()
{
    vector<string> petList = {"Dog"};
    petList.push_back("Cat");
    petList.push_back("Fish");

    petList.insert(petList.begin() + 1, "Golden Bandicoot");
    petList.erase(petList.begin() + 0);

    for (string item : petList)
    {
        cout << item << " ";
    }
}